k8s/demo/common/xui/xui-tc.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
```

Comment:
Do u know demo doesn't work same as other environments with current setup?

Reply:
I only know we can use Product chart or Microservice chart in demo, but didnt know the actual setup will be different in flux config. Can you point me in the right direction to setup demo env for using microservice chart? thx

Reply:
I mean environment itself is setup differently. For example, it doesn't have connectivity to existing postgres databases. so, ur config won't wait.

Reply:
so what we need to do if we would like to connect to a PaaS postgre instance? i.e  xui-terms-and-conditions-demo . Or the demo environment doesnt support connect to any PaaS DB?

Reply:
It doesn't support connecting to any Paas DB.